src/main/java/teammates/common/datatransfer/FeedbackContributionQuestionDetails.java

```diff
@@ -395,7 +395,8 @@ public String getQuestionResultStatisticsCsv(
         String csv = "";
         
         //Header
-        csv += "In the points given below, an equal share is equal to 100 points. ";
+        csv += Const.FEEDBACK_CONTRIBUTION_HEADER_BEFORE_COMMA;
```

Comment:
The variable name does not make much sense. Why use a constant here? Can this be avoided?